### DISCUSSION TITLE
Quit app/game when user chooses to quit session

### DISCUSSION THREAD

**Discussion by quikbeam** [LizardByte/community#124] [Sunshine] [answered]
Gamestream through Moonlight quits game automatically when you quit the session 

Currently doing this with undo commands ex: taskkill /im nioh2.exe /f /fi "STATUS eq RUNNING"

**Comment by ReenigneArcher** [comment, marked as answer]
Use "cmd" instead of detached.

**Reply by quikbeam** [reply]
Tried "command" and it works even better thanks!
